DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/bridging/test_bridging_bum_traffic_bridge_with_rif.py

```diff
@@ -0,0 +1,469 @@
+            assert tgen_utils_get_loss(row) == 100.000, \
+                f"Verify that traffic from swp1 to swp2 not forwarded.\n"
+        if row["Traffic Item"] == "IPv4_DCHP_BC" and row["Rx Port"] == tg_ports[1]:
+            assert tgen_utils_get_loss(row) == 0.000, \
+                f"Verify that traffic from swp1 to swp2 forwarded.\n"
+        if row["Traffic Item"] == "IPv4_ReservedMC" and row["Rx Port"] == tg_ports[1]:
+            assert tgen_utils_get_loss(row) == 0.000, \
+                f"Verify that traffic from swp1 to swp2 forwarded.\n"
+        if row["Traffic Item"] == "IPv4_AllSysInSubnet" and row["Rx Port"] == tg_ports[1]:
+            assert tgen_utils_get_loss(row) == 0.000, \
+                f"Verify that traffic from swp1 to swp2 forwarded.\n"
+        if row["Traffic Item"] == "IPv4_AllRoutersInSubnet" and row["Rx Port"] == tg_ports[1]:
+            assert tgen_utils_get_loss(row) == 0.000, \
+                f"Verify that traffic from swp1 to swp2 forwarded.\n"
+        if row["Traffic Item"] == "IPv4_OSPFIGP" and row["Rx Port"] == tg_ports[1]:
+            assert tgen_utils_get_loss(row) == 0.000, \
+                f"Verify that traffic from swp1 to swp2 forwarded.\n"
+        if row["Traffic Item"] == "IPv4_RIP2" and row["Rx Port"] == tg_ports[1]:
+            assert tgen_utils_get_loss(row) == 0.000, \
+                f"Verify that traffic from swp1 to swp2 forwarded.\n"
+        if row["Traffic Item"] == "IPv4_EIGRP" and row["Rx Port"] == tg_ports[1]:
+            assert tgen_utils_get_loss(row) == 0.000, \
+                f"Verify that traffic from swp1 to swp2 forwarded.\n"
+        if row["Traffic Item"] == "IPv4_DHCPServerRelay" and row["Rx Port"] == tg_ports[1]:
+            assert tgen_utils_get_loss(row) == 0.000, \
+                f"Verify that traffic from swp1 to swp2 forwarded.\n"
+        if row["Traffic Item"] == "IPv4_VRRP" and row["Rx Port"] == tg_ports[1]:
+            assert tgen_utils_get_loss(row) == 0.000, \
+                f"Verify that traffic from swp1 to swp2 forwarded.\n"
+        if row["Traffic Item"] == "IPv4_IGMP" and row["Rx Port"] == tg_ports[1]:
+            assert tgen_utils_get_loss(row) == 0.000, \
+                f"Verify that traffic from swp1 to swp2 forwarded.\n"
+        if row["Traffic Item"] == "IPV4_BGP" and row["Rx Port"] == tg_ports[1]:
+            assert tgen_utils_get_loss(row) == 100.000, \
+                f"Verify that traffic from swp1 to swp2 not forwarded.\n"
+
+    await tcpdump
+    print(("TCPDUMP: packets=%s" % tcpdump.result()))
+    data = tcpdump.result()
+
+    count_of_packets = re.findall(r"(\d+) packets (captured|received|dropped)", data)
+    assert int(count_of_packets[0][0]) > 0, f"Verify that packets captured.\n"
+    assert int(count_of_packets[1][0]) > 0, f"Verify that packets received by filter.\n"
+    assert int(count_of_packets[2][0]) == 0, f"Verify that packets dropped by kernel.\n"
```

Comment:
Don't expect that your matches will be in order. It is better to use a loop:
```suggestion
    count_of_packets = re.findall(r"(\d+) packets (captured|received|dropped)", data)
    for count, type in count_of_packets:
        if type == "captured" or type == "received":
            assert int(count) > 0, f"Verify that packets are captured and received.\n"
        if type == "dropped":
            assert int(count) == 0, f"Verify that packets are dropped by kernel.\n"
```